packages/logs/src/domain/logsCollection/report/reportCollection.ts

```diff
@@ -18,18 +18,16 @@ const LogStatusForReport = {
   [RawReportType.deprecation]: StatusType.warn,
 }
 
-export function startReportCollection(configuration: LogsConfiguration, sender: Sender) {
+export function startReportCollection(configuration: LogsConfiguration, lifeCycle: LifeCycle) {
   const reportObservable = initReportObservable(configuration.forwardReports)
   const reportSubscription = reportObservable.subscribe(logReport)
 
   function logReport(report: RawReport) {
     let message = report.message
-    const messageContext: Partial<LogsEvent> = {
-      origin: ErrorSource.REPORT,
-    }
-    const logStatus = LogStatusForReport[report.type]
-    if (logStatus === StatusType.error) {
-      messageContext.error = {
+    const status = LogStatusForReport[report.type]
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** we could inline this function to avoid to name it‏